sys/include/suit/handlers.h

```diff
@@ -193,6 +193,12 @@ int suit_handle_manifest_structure_bstr(suit_manifest_t *manifest,
                                         const suit_manifest_handler_t *handlers,
                                         size_t handlers_len);
 
+uint16_t suit_param_ref_to_cbor(suit_manifest_t *manifest,
+                                suit_param_ref_t *ref,
+                                nanocbor_value_t *val);
+void suit_param_cbor_to_ref(suit_manifest_t *manifest,
+                           suit_param_ref_t *ref,
+                           nanocbor_value_t *val);
```

Comment:
Doxygen missing.

Reply:
Huh, why is Murdock not screaming about this. Thanks for spotting!

Reply:
Added the doxygen

Reply:
Thanks!